static/app/components/events/interfaces/spans/spanTreeModel.tsx

```diff
@@ -34,9 +35,9 @@ export const MIN_SIBLING_GROUP_SIZE = 5;
 
 class SpanTreeModel {
   api: Client;
+  span: SpanType;
 
   // readonly state
-  span: Readonly<SpanType>;
```

Comment:
I had to change `span` so it wasn't Readonly in order to modify the timestamps, not sure if this is a safe change though.